packages/gitbook/src/components/DocumentView/Table/table.module.css

```diff
@@ -27,33 +27,33 @@
 }
 
 .columnHeader {
-    @apply text-sm font-medium py-3 px-4 text-tint-900;
+    @apply text-sm font-medium py-2 px-4 text-tint-900;
 }
 
 :global(.dark) .columnHeader {
     @apply text-white;
 }
 
 .row {
-    @apply flex border-tint-700/2;
+    @apply flex border-tint-700/2 hover:bg-tint-700/2 transition-colors;
```

Comment:
Ideally I'd apply `hover:bg-tint-700/1` `dark:hover:bg-tint-700/2` so the dark mode hover color is different than the light-mode, but tailwind doesn't agree. Open to suggestions, otherwise ill look into it later and fix after.